packages/mds-transactions-service/tests/service.spec.ts

```diff
@@ -121,22 +122,67 @@ describe('Transaction Service Tests', () => {
     }
   })
 
+  function* bulkTransactions(): Generator<TransactionDomainModel> {
+    let timestamp = Date.now() - 20 * 1000
+    for (let i = 0; i < 20; i++) {
+      yield {
+        transaction_id: uuid(),
+        provider_id,
+        device_id,
+        timestamp,
+        amount: 100, // "I'd buy THAT for a dollar!"
+        fee_type: 'base_fee',
+        receipt
+      }
+      timestamp += 1000
+    }
+  }
+
+  it('Bulk Post 20 Transactions', async () => {
+    const transactions = await TransactionServiceClient.createTransactions([...bulkTransactions()])
+    expect(transactions[0].device_id).toEqual(device_id)
+    // TODO more checks
+  })
+
   it('Get All Transactions', async () => {
-    const transactions = await TransactionServiceClient.getTransactions({})
-    expect(transactions.length).toEqual(1)
+    const { transactions } = await TransactionServiceClient.getTransactions({})
+    expect(transactions.length).toEqual(10)
     const [transaction] = transactions
     expect(transaction.transaction_id).toEqual(transaction_id)
```

Comment:
I'm going to wack this, it's due to a side-effect and doesn't _really_ relate to the purpose of this test (which is to demonstrate that the default limit works)